rts/at/bridge/src/main/java/org/jboss/narayana/rest/bridge/inbound/InboundBridge.java

```diff
@@ -212,11 +222,15 @@ private void enlist(final InboundBridge inboundBridge) {
      *
      * @return
      */
-    private Transaction getTransaction() {
+    private Transaction getTransaction(boolean useContainerToLookup) {
```

Comment:
I just raised this to check if we do really need the resume handling: https://github.com/jbosstm/narayana/pull/1139